From: Andrea Bolognani <eof@kiyuko.org>
Date: Sat, 13 Jan 2024 16:06:58 +0100
Subject: Set DesktopNames in xsession configuration

Login managers that use xsession files will parse this key and
set $XDG_CURRENT_DESKTOP accordingly. That environment variable,
in turn, will influence other components that can exhibit
desktop-specific behavior.

(cherry picked from commit 4b0a23c7017d7ffc4b5436e58b738e45d466ed64)

Forwarded: not-needed
Origin: https://github.com/conformal/spectrwm/commit/4b0a23c7017d7ffc4b5436e58b738e45d466ed64
---
 linux/spectrwm.desktop | 1 +
 1 file changed, 1 insertion(+)

diff --git a/linux/spectrwm.desktop b/linux/spectrwm.desktop
index b9e4ad4..9a0576e 100644
--- a/linux/spectrwm.desktop
+++ b/linux/spectrwm.desktop
@@ -4,4 +4,5 @@ Comment=The spectrwm window manager
 Type=Application
 Exec=spectrwm
 TryExec=spectrwm
+DesktopNames=spectrwm;
 Keywords=tiling;window;manager;wm;
